src/radical/pilot/session.py

```diff
@@ -800,13 +579,20 @@ def list_resources(self):
         description.  Not that resource aliases won't be listed.
         '''
 
-        return sorted(self._resource_configs.keys())
+        resources = list()
+        for domain in self._rcfgs:
+            if domain == 'aliases':
+                continue
+            for host in self._rcfgs:
+                resources.append('%s.%s' % (domain, host))
+
+        return sorted(resources)
 
 
     # -------------------------------------------------------------------------
     #
     def add_resource_config(self, resource_config):
-        """Adds a new :class:`radical.pilot.ResourceConfig` to the PilotManager's
+        '''Adds a new :class:`radical.pilot.ResourceConfig` to the PilotManager's
```

Comment:
Along all the file we use ` '''Word ` and ` '''/n ` inconsistently.

Reply:
Alas we do.  I converted many already (also from `"""` to `'''`, which was used inconsistently), but I am sure many remain...